feat: alert on chromium-based browsers

rodcoffani opened the pull request.

rodcoffani committed:
feat: add alert on chromium browsers

- add alert when opening MS Office files
- only on chromium-based browsers
- closed stated saved on localStorage

diocas commented on packages/web-app-external/src/App.vue:
```diff
@@ -91,6 +91,13 @@ export default defineComponent({
+    },
+    chromiumBased() {
+      // @ts-ignore
+      return !!window.chrome || navigator.userAgent.includes('Chrome')
```
Does this cover all chromium versions? Doesn't like so (e.g. Safari). Maybe it would be better to check that is not firefox instead.

rodcoffani replied:
All chromium browsers should have the `window.chrome` function (which usually doesn't exist on the Window API).
But changing the fallback could be a good choice.

diocas replied:
Ah, understood!

diocas reviewed (changes requested):
Can you add a screenshot of how it looks?

Also, it's missing the subscription of the MS office event that says everything is fine. On those situations, I would just close the banner.

rodcoffani commented:
Preview:
![image](https://github.com/user-attachments/assets/45558d45-080a-4aab-865f-f424f7b91b77)

rodcoffani committed:
fix: update browser checking

rodcoffani committed:
fix: remove logs

rodcoffani committed:
fix: successful load

- remove alert when it has a successful load
- moved code to `try` instead of `finally`
